src/common/modules/navigations/gnb/modules/gnb-menu/modules/dashboard-recent-favorite/modules/GNBDashboardFavorite.vue

```diff
@@ -0,0 +1,339 @@
+                    items = state.favoriteProjects;
+                    label = i18n.t('COMMON.GNB.FAVORITES.ALL_PROJECTS');
+                }
+                if (state.showAllType === SUGGESTION_TYPE.CLOUD_SERVICE) {
+                    items = state.favoriteCloudServiceItems;
+                    label = i18n.t('COMMON.GNB.FAVORITES.ALL_CLOUD_SERVICES');
+                }
+                return [
+                    {
+                        name: 'title', type: 'header', label, itemType: state.showAllType,
+                    },
+                    ...items,
+                ];
+            }),
+            //
+            cloudServiceTypes: computed<CloudServiceTypeReferenceMap>(() => store.getters['reference/cloudServiceTypeItems']),
+            projects: computed<ProjectReferenceMap>(() => store.getters['reference/projectItems']),
+            projectGroups: computed<ProjectGroupReferenceMap>(() => store.getters['reference/projectGroupItems']),
+            //
+            favoriteMenuItems: computed<FavoriteItem[]>(() => convertMenuConfigToReferenceData(
+                store.state.favorite.menuItems,
+                store.getters['display/allMenuList'],
+            )),
+            favoriteCloudServiceItems: computed<FavoriteItem[]>(() => {
+                const isUserAccessible = isUserAccessibleToMenu(MENU_ID.ASSET_INVENTORY_CLOUD_SERVICE, store.getters['user/pagePermissionList']);
+                return isUserAccessible ? convertCloudServiceConfigToReferenceData(
+                    store.state.favorite.cloudServiceItems,
+                    state.cloudServiceTypes,
+                ) : [];
+            }),
+            favoriteProjects: computed<FavoriteItem[]>(() => {
+                const isUserAccessible = isUserAccessibleToMenu(MENU_ID.PROJECT, store.getters['user/pagePermissionList']);
+                if (!isUserAccessible) return [];
+                const favoriteProjectItems = convertProjectConfigToReferenceData(store.state.favorite.projectItems, state.projects);
+                const favoriteProjectGroupItems = convertProjectGroupConfigToReferenceData(store.state.favorite.projectGroupItems, state.projectGroups);
+                return [...favoriteProjectGroupItems, ...favoriteProjectItems];
+            }),
+        });
+
+        /* Util */
+        const getItemLength = (type: SuggestionType): number => {
+            if (type === SUGGESTION_TYPE.MENU) return state.favoriteMenuItems.length;
+            if (type === SUGGESTION_TYPE.PROJECT) return state.favoriteProjects.length;
+            if (type === SUGGESTION_TYPE.CLOUD_SERVICE) return state.favoriteCloudServiceItems.length;
+            return 0;
+        };
+
+        /* Event */
+        const handleClickMenuButton = (type: SuggestionType) => {
+            if (type === SUGGESTION_TYPE.PROJECT) {
+                SpaceRouter.router.replace({
+                    name: PROJECT_ROUTE._NAME,
+                });
```

Comment:
혹시 여기 프로젝트 페이지에서 이 메뉴 버튼을 눌렀을 때 에러가 나는지 봐주시겠어요?
동일한 페이지로 또 이동할 때 router 에러가 발생했던 것 같아서...
만약 에러가 난다면 이렇게 예외처리 해주어야 합니다!

```suggestion
                SpaceRouter.router.replace({
                    name: PROJECT_ROUTE._NAME,
                }).catch(() => {});;
```

Reply:
아하 넵 내부 구현할때 같이 적용해 둘게욥!